src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (numA) => {
+    count++;
```

Comment:
It's not the right place for incrementing the counter. Let's put it inside the inner function